spinalcordtoolbox/scripts/sct_extract_metric.py

```diff
@@ -357,7 +357,7 @@ def main(fname_data, path_label, method, slices, levels, fname_output, labels_us
     labels = np.concatenate(labels_tmp[:], 3)  # labels: (x,y,z,label)
     # Load vertebral levels
     if vertebral_levels:
-        im_vertebral_labeling = Image(fname_vertebral_labeling).change_orientation("RPI")
+        im_vertebral_labeling = fname_vertebral_labeling        
     else:
         im_vertebral_labeling = None
```

Comment:
maybe we can simply get rid of that, and pass `fname_vertebral_labeling` into the function `extract_metric()` below. We would need to make sure the argument is "None" if not explicitly set by the user in the CLI parser. 

also, just for your information, we have [conventions for variable names](https://github.com/neuropoly/spinalcordtoolbox/wiki/Programming%3A-Variables#variables), and "im_" corresponds to Image() objects (so in this case, the proposed change would go against the convention).

Reply:
Resolved by e9ddb86b718d7fd5ef5fb04078358539e5e9b930